.travis.yml

```diff
@@ -8,8 +8,9 @@ osx_image: xcode6.4
 
 env:
   matrix:
-    
+
     - CONDA_PY=27
+    - CONDA_PY=34
```

Comment:
Conda-forge does not support `Python 3.4` and manual edits to the feedstock is discourage.
The main reason is the load in the CIs as we cannot afford to build 3 version for Python 3 (3.4, 3.5, and 3.6).

Please consider re-rendering this with latest conda-smithy to remove the addition matrix items. (Note that the package you created with this PR will remain in the channel, so it is a win-win :wink:)